explorer/app/components/common/StaticImage/staticImage.tsx

```diff
@@ -27,8 +23,13 @@ export const StaticImage = ({
   src,
   width,
 }: StaticImageProps): JSX.Element => {
-  return (
-    <img alt={alt} height={height} src={`${BASE_PATH}${src}`} width={width} />
-  );
+  const router = useRouter();
+
+  // If src is a image in base64, don't add the basePath
+  const srcValue = src?.startsWith("data:image/")
```

Comment:
Nih svg image is the only one that has the src as a base64 image. I have no idea why. Try to convert it into different formats but didn't work. So I'm just addressing base64 images here

Reply:
Yes, there are a few others. I think it does this to small ones on the build.